src/rcw/pdf.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@nestjs/common';
+import { exec as execCb } from 'child_process';
+import { promisify } from 'util';
+
+const exec = promisify(execCb);
+
+@Injectable()
+export class ExecService {
+  async pdfConvertorCommand(filePath, outputPath): Promise<void> {
+    try {
+      const { stdout, stderr } = await exec(`gs -dPDFA -dBATCH -dNOPAUSE -sProcessColorModel=DeviceCMYK -sDEVICE=pdfwrite -sPDFACompatibilityPolicy=1 -sOutputFile="${outputPath}" "${filePath}"`);
+      console.log(`stdout: ${stdout}`);
```

Comment:
Let's move these logs to nestJS's built in logger.

Reply:
Done